codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientDecorator.kt

```diff
@@ -65,7 +66,7 @@ sealed class FluentClientSection(name: String) : Section(name) {
     /** Write custom code into an operation fluent builder's impl block */
     data class FluentBuilderImpl(
         val operationShape: OperationShape,
-        val operationErrorType: RuntimeType,
+        val operationErrorType: Symbol,
```

Comment:
Problem is, we don't have a `RuntimeType` when we move to determining modules based on the shape and symbol provider, so it isn't really feasible to preserve this being a `RuntimeType`. `Symbol` is kind of the lowest common denominator since everything can convert into it, and it is ultimately the renderable type.

I don't think we really need a `RuntimeType` here though. We only use this for rendering.